packages/components/src/components/colors.ts

```diff
@@ -46,49 +47,94 @@ export const colorDictionary = {
   grey: '#1E2326',
   greyLight: '#595C5F',
   greyHover: '#F8F8F8',
-  grey800: '#1E2326',
-  grey300: '#F1F2F3',
-  grey600: '#909397',
+  // grey800: '#1E2326',
+  // grey300: '#F1F2F3',
+  // grey600: '#909397',
   greyInfo: '#EFEFEF',
   indigo500: '#4972BB',
-  indigo600: '##42639C',
+  indigo600: '#42639C',
   yellow500: '#EDC55E'
 }
 
 export const gradients = {
   gradientNightshade:
     'background: linear-gradient(180deg, #42506B 0%, #485F88 100%)',
   gradientSkyDark:
-    'background: linear-gradient(180deg, #3C55A3 0%, #4C68C1 100%)',
+    'background: linear-gradient(180deg, #3C55A3 0%, #4C68C1 100%)', // replace and delete
   gradientSkyLight:
-    'background: linear-gradient(180deg, #6291CD 0%, #AACAF3 100%)',
+    'background: linear-gradient(180deg, #6291CD 0%, #AACAF3 100%)', // replace and delete
   gradientBabyShade:
-    'background: linear-gradient(180deg, #477cd7 0%, #5c91eb 100%);',
+    'background: linear-gradient(180deg, #477cd7 0%, #5c91eb 100%);', // replace and delete
   gradientGreyShade:
     'background: linear-gradient(180deg, #FFFFFF 0%, #F2F3F4 100%);'
 }
 
 export const shadows = {
-  mistyShadow: 'box-shadow: 0px 2px 6px rgba(53, 67, 93, 0.32)',
-  thickShadow: 'box-shadow: 0px 2px 8px rgba(53, 67, 93, 0.54)'
+  mistyShadow: 'box-shadow: 0px 2px 6px rgba(53, 67, 93, 0.32)', // rename lightShadow - dropdowns selects
+  thickShadow: 'box-shadow: 0px 2px 8px rgba(53, 67, 93, 0.54)' // rename heavyShadow - models
 }
 
 export const colors = {
-  primary: colorDictionary.blueDeepSea,
-  secondary: colorDictionary.blueBabyBaby,
-  tertiary: colorDictionary.blueDeepSea,
-  error: colorDictionary.redDanger,
-  errorHover: colorDictionary.redDangerDark,
-  warning: colorDictionary.orangeAmber,
-  copy: colorDictionary.blackStormy,
-  placeholder: colorDictionary.greyDarkSteel,
-  background: colorDictionary.greyPearl,
-  disabled: colorDictionary.greyGrey,
-  success: colorDictionary.greenPeaPea,
-  successHover: colorDictionary.greenPeaPeaDark,
-  white: colorDictionary.white,
-  black: colorDictionary.black,
-  focus: colorDictionary.yellowFocus,
+  // Pallete
+  primary: '#4972BB', // indigo
+  secondary: '#4A8AD7', // blue
+  tertiary: '#cccfd0', // grey
+
+  purple: '#8049B7', // in progress
+  orange: '#F1B162', // ready for review
+  red: '#d53f3f', // requires updates
+  green: '#49b78d', // registered
+  blue: '#4A8AD7', // certified
+  teal: '#4CC1BA', // charts
+
+  // Hover states (note: these should be removed in future and use Darken https://polished.js.org/docs/#darken )
+  indigoDark: '#42639C',
+
+  // Status
+  success: '#49b78d', // rename to positive - green buttons, lighten to make light green backgrounds
+  successHover: '#409977', // remove and use darken of success/positive colour for hover states
```

Comment:
@jpye-finch are you going to attend to these commented tasks in this PR?  If not, are these comments ok to stay in the code?